plot/plot.app.js

```diff
@@ -191,26 +188,44 @@ function loadModule() {
                       if (isNaN(formated_data)) {
                         return data;
                       }
-                      return parseInt(formated_data.toLocaleString('en-US'));
+                      return  formated_data.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,')
                     } catch (e) {
                       return data;
                     }
                   }
                   else {
-                    return data
+                    return data.replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
                   }
                 }
 
                 return {
-                    getData: function () {
+                    getData: function (config) {
                         var plots = [];
                         var count = 0;
-                        plotConfig.plots.forEach(function () {
-                            plots.push({id: count, loaded: false});
-                            count+=1
-                        });
+                        var plotConfig = plotConfigs[config];
+                        var message = "";
+                        var error;
+
+                        if (plotConfig == undefined) {
+                          if (config) {
+                            message = "Invalid config parameter in the url";
+                            error = new Errors.CustomError("Invalid config", message);
```

Comment:
Do we want this to error out? Or should we show a warning to the user and default to the "*" context (if defined)?

I think the case when the config doesn't match and "*" is not defined, should be when an error is thrown.

Reply:
This was done for the case if the person specifies `config='something'` and that config is not present, then show the error( Instead of using the default config). 
If the config parameter is not defined in the query then use the default config.   
I'll check with Hongsuda about this.

Reply:
@hongsudt Need your thoughts on this.

Reply:
@jrchudy I had a chat with hongsuda, and she agrees with the way I implemented it.